features/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Used only for running tests using tox, see ../tox.ini
+FROM patroni
+
+WORKDIR $PGHOME
```

Comment:
is it actually required? or you wanted to create `src` directory inside `$PGHOME`? 
unfortunately, it is anyway not set here (it is only ARG in the original Dockerfile, not ENV)